### PR TITLE
Comment out ellipsis in code blocks

### PR DESCRIPTION
**Pull request by OnkarRuikar**
Ellipsis in markdown code blocks are syntax errors.

In order to have smooth transition to Prettier and ESLint the PR comments out such bare ellipses. Refer the https://github.com/mdn/content/pull/18171#issuecomment-1179670484 for more context.

### CODE REVIEW

File: files/en-us/web/api/mediarecorder/resume_event/index.md [hunk 1]
```diff
@@ -59,7 +59,7 @@ A generic {{domxref("Event")}}.
     // recording being resumed
   }
 
-...
+// …
```
Comment (teoli2003) [inline files/en-us/web/api/mediarecorder/resume_event/index.md:62]:
```suggestion
```

File: files/en-us/web/api/mediarecorder/start/index.md [hunk 2]
```diff
@@ -100,14 +100,14 @@ handler to respond to these errors.
 ## Examples
 
 ```js
-...
+// …
```
Comment (teoli2003) [inline files/en-us/web/api/mediarecorder/start/index.md:104]:
```suggestion
```

File: files/en-us/web/api/mediarecorder/start/index.md [hunk 3]
```diff
@@ -100,14 +100,14 @@ handler to respond to these errors.
...
 
   record.onclick = function() {
     mediaRecorder.start();
     console.log("recorder started");
   }
```
Comment (teoli2003) [inline files/en-us/web/api/mediarecorder/start/index.md:109]:
```suggestion
```

File: files/en-us/web/api/mediarecorder/start/index.md [hunk 4]
```diff
@@ -100,14 +100,14 @@ handler to respond to these errors.
...
 
-...
+// …
```
Comment (teoli2003) [inline files/en-us/web/api/mediarecorder/start/index.md:110]:
```suggestion
```

File: files/en-us/web/api/mediarecorder/start_event/index.md [hunk 5]
```diff
@@ -38,7 +38,7 @@ A generic {{domxref("Event")}}.
 ## Example
 
 ```js
-...
+// …
```
Comment (teoli2003) [inline files/en-us/web/api/mediarecorder/start_event/index.md:42]:
```suggestion
```

File: files/en-us/web/api/mediarecorder/start_event/index.md [hunk 6]
```diff
@@ -50,7 +50,7 @@ A generic {{domxref("Event")}}.
     // recording being started
   }
```
Comment (teoli2003) [inline files/en-us/web/api/mediarecorder/start_event/index.md:52]:
```suggestion
```

File: files/en-us/web/api/mediarecorder/start_event/index.md [hunk 7]
```diff
@@ -50,7 +50,7 @@ A generic {{domxref("Event")}}.
...
 
-...
+// …
```
Comment (teoli2003) [inline files/en-us/web/api/mediarecorder/start_event/index.md:53]:
```suggestion
```

File: files/en-us/web/api/mediarecorder/state/index.md [hunk 8]
```diff
@@ -32,7 +32,7 @@ A string containing one of the following values:
 ## Examples
 
 ```js
-...
+// …
```
Comment (teoli2003) [inline files/en-us/web/api/mediarecorder/state/index.md:36]:
```suggestion
```

File: files/en-us/web/api/mediarecorder/state/index.md [hunk 9]
```diff
@@ -41,7 +41,7 @@ A string containing one of the following values:
     console.log("recorder started");
   }
```
Comment (teoli2003) [inline files/en-us/web/api/mediarecorder/state/index.md:43]:
```suggestion
```

File: files/en-us/web/api/mediarecorder/state/index.md [hunk 10]
```diff
@@ -41,7 +41,7 @@ A string containing one of the following values:
...
 
-...
+// …
```
Comment (teoli2003) [inline files/en-us/web/api/mediarecorder/state/index.md:44]:
```suggestion
```

File: files/en-us/web/api/mediarecorder/stop/index.md [hunk 11]
```diff
@@ -53,14 +53,14 @@ is "inactive" — it makes no sense to stop media capture if it is already stopp
 ## Examples
 
 ```js
-...
+// …
```
Comment (teoli2003) [inline files/en-us/web/api/mediarecorder/stop/index.md:57]:
```suggestion
```

File: files/en-us/web/api/mediarecorder/stop/index.md [hunk 12]
```diff
@@ -53,14 +53,14 @@ is "inactive" — it makes no sense to stop media capture if it is already stopp
...
 
   stop.onclick = function() {
     mediaRecorder.stop();
     console.log("recorder stopped, data available");
   }
```
Comment (teoli2003) [inline files/en-us/web/api/mediarecorder/stop/index.md:62]:
```suggestion
```

File: files/en-us/web/api/mediarecorder/stop/index.md [hunk 13]
```diff
@@ -53,14 +53,14 @@ is "inactive" — it makes no sense to stop media capture if it is already stopp
...
 
-...
+// …
```
Comment (teoli2003) [inline files/en-us/web/api/mediarecorder/stop/index.md:63]:
```suggestion
```

File: files/en-us/web/api/mediarecorder/stop_event/index.md [hunk 14]
```diff
@@ -40,7 +40,7 @@ A generic {{domxref("Event")}}.
 ## Example
 
 ```js
-...
+// …
```
Comment (teoli2003) [inline files/en-us/web/api/mediarecorder/stop_event/index.md:44]:
```suggestion
```

File: files/en-us/web/api/mediarecorder/stop_event/index.md [hunk 15]
```diff
@@ -57,7 +57,7 @@ A generic {{domxref("Event")}}.
     chunks.push(e.data);
   }
```
Comment (teoli2003) [inline files/en-us/web/api/mediarecorder/stop_event/index.md:59]:
```suggestion
```

File: files/en-us/web/api/mediarecorder/stop_event/index.md [hunk 16]
```diff
@@ -57,7 +57,7 @@ A generic {{domxref("Event")}}.
...
 
-...
+// …
```
Comment (teoli2003) [inline files/en-us/web/api/mediarecorder/stop_event/index.md:60]:
```suggestion
```

File: files/en-us/web/api/mediarecorder/stream/index.md [hunk 17]
```diff
@@ -41,7 +41,7 @@ if (navigator.getUserMedia) {
            const myStream = mediaRecorder.stream;
            console.log(myStream);
```
Comment (teoli2003) [inline files/en-us/web/api/mediarecorder/stream/index.md:43]:
```suggestion
```

File: files/en-us/web/api/mediarecorder/stream/index.md [hunk 18]
```diff
@@ -41,7 +41,7 @@ if (navigator.getUserMedia) {
...
 
-...
+// …
```
Comment (teoli2003) [inline files/en-us/web/api/mediarecorder/stream/index.md:44]:
```suggestion
```

File: files/en-us/web/api/mediarecorder/warning_event/index.md [hunk 19]
```diff
@@ -28,13 +28,13 @@ A function reference.
 ## Examples
 
 ```js
-...
+// …
```
Comment (teoli2003) [inline files/en-us/web/api/mediarecorder/warning_event/index.md:32]:
```suggestion
```

File: files/en-us/web/api/mediarecorder/warning_event/index.md [hunk 20]
```diff
@@ -28,13 +28,13 @@ A function reference.
...
 
   mediaRecorder.onwarning = function(e) {
     console.log("A warning has been raised: " + e.message);
   }
```
Comment (teoli2003) [inline files/en-us/web/api/mediarecorder/warning_event/index.md:36]:
```suggestion
```

File: files/en-us/web/api/mediarecorder/warning_event/index.md [hunk 21]
```diff
@@ -28,13 +28,13 @@ A function reference.
...
 
-...
+// …
```
Comment (teoli2003) [inline files/en-us/web/api/mediarecorder/warning_event/index.md:37]:
```suggestion
```

**Review by teoli2003 — CHANGES_REQUESTED** [review]
A lot of nit (matter of taste) where I could just remove the ellipses (and givent the GitHub UI, I had to make 3 suggestions each time).

Some more serious errors in the code that we need to correct immediately.

**Comment by OnkarRuikar** [comment]
Thanks! Ellipses have been used lavishly. 😥

I've fixed the code bugs. And resolved conflicts with the other PR.

**Review by teoli2003 — APPROVED** [review]
Yes, there would be many more to remove. But that's not the goal, the goal is to have (more) syntactically correct code blocks to ease cutting and pasting and help us run Prettier on code snippets in the future.

(But sometimes it is so ugly that I can't resist)

**Pull request merged**